.github/workflows/wheel_linux_aarch64_cuda.yml

```diff
@@ -67,7 +67,7 @@ jobs:
 
           # Python build settings
           CIBW_BEFORE_BUILD: |
-            python -m pip install ninja cmake~=3.24.3 auditwheel~=5.0 custatevec-cu${{ matrix.cuda_version }}
+            python -m pip install ninja cmake~=3.27.0 auditwheel~=5.0 custatevec-cu${{ matrix.cuda_version }}
```

Comment:
I'm all in for updating packages, but I'm curious if you needed to do that for any reason, like incompatibilities.

Reply:
The latest PB11 documents that some supports require a newer CMake, with 3.27 a minimum for the new Python finder utility, which defers mostly to CMake's internals instead of a custom implementation.